services/fxa/serverless.yml

```diff
@@ -2,131 +2,37 @@
-    PROJECT_NAME: ${env:PROJECT_NAME}
-    BRANCH: ${env:BRANCH}
-    REVISION: ${env:REVISION}
-    VERSION: ${env:VERSION}
-    REMOTE_ORIGIN_URL: ${env:REMOTE_ORIGIN_URL}
-    LOG_LEVEL: ${env:LOG_LEVEL}
-    NEW_RELIC_ACCOUNT_ID: ${env:NEW_RELIC_ACCOUNT_ID}
-    NEW_RELIC_TRUSTED_ACCOUNT_ID: ${env:NEW_RELIC_TRUSTED_ACCOUNT_ID}
-    NEW_RELIC_SERVERLESS_MODE_ENABLED: ${env:NEW_RELIC_SERVERLESS_MODE_ENABLED}
-    NEW_RELIC_DISTRIBUTED_TRACING_ENABLED: ${env:NEW_RELIC_DISTRIBUTED_TRACING_ENABLED}
-    PROFILING_ENABLED: ${env:PROFILING_ENABLED}
-    USER_TABLE:
-      Ref: 'Users'
-    EVENT_TABLE:
-      Ref: 'Events'
-    DELETED_USER_TABLE:
-      Ref: 'DeletedUsers'
-  tags:
-    cost-center: 1440
-    project-name: subhub
-    project-desc: payment subscription REST api for customers
-    project-email: subhub@mozilla.com
-    deployed-by: ${env:DEPLOYED_BY}
-    deployed-env: ${env:DEPLOYED_ENV}
-    deployed-when: ${env:DEPLOYED_WHEN}
-    deployed-method: serverless
-    sources: https://github.com/mozilla/subhub
-    urls: prod.fxa.mozilla-subhub.app/v1
-    keywords: subhub:subscriptions:flask:serverless:swagger
-    branch: ${env:BRANCH}
-    revision: ${env:REVISION}
-    version: ${env:VERSION}
-  stackTags:
-    service: ${self:service}
-  iamRoleStatements:
-    - Effect: Allow
-      Action:
-        - 'dynamodb:Query'
-        - 'dynamodb:Scan'
-        - 'dynamodb:GetItem'
-        - 'dynamodb:PutItem'
-        - 'dynamodb:UpdateItem'
-        - 'dynamodb:DeleteItem'
-        - 'dynamodb:DescribeTable'
-        - 'dynamodb:CreateTable'
-      Resource:
-        - { 'Fn::GetAtt': ['Users', 'Arn'] }
-        - { 'Fn::GetAtt': ['Events', 'Arn'] }
-        - { 'Fn::GetAtt': ['DeletedUsers', 'Arn']}
-    - Effect: Allow
-      Action:
-        - 'secretsmanager:GetSecretValue'
-      Resource:
-        - 'Fn::Join': [':', ['arn:aws:secretsmanager', Ref: AWS::Region, Ref: AWS::AccountId, 'secret:${self:provider.stage}/*']]
-    - Effect: Allow
-      Action:
-        - logs:CreateLogGroup
-        - logs:CreateLogStream
-        - logs:PutLogEvents
-      Resource:
-        - 'Fn::Join': [':', ['arn:aws:logs', Ref: AWS::Region, Ref: AWS::AccountId, 'log-group:/aws/lambda/*:*:*']]
-    - Effect: Allow
-      Action:
-        - kms:Decrypt
-      Resource:
-        - 'Fn::Join': [':', ['arn:aws:kms', Ref: AWS::Region, Ref: AWS::AccountId, 'alias/*']]
-        - 'Fn::Join': [':', ['arn:aws:kms', Ref: AWS::Region, Ref: AWS::AccountId, 'key/*']]
-    - Effect: Allow
-      Action:
-        - 'xray:PutTraceSegments'
-        - 'xray:PutTelemetryRecords'
-      Resource:
-        - '*'
-    - Effect: Allow
-      Action:
-        - sns:Publish
-      Resource:
-        - 'Fn::Join': [':', ['arn:aws:sns', Ref: AWS::Region, Ref: AWS::AccountId, '${self:provider.stage}-fxa-event-data']]
-  resourcePolicy: ${self:custom.resourcePolicies.${self:custom.access.${self:provider.stage}}}
-
 package:
+  individually: false
```

Comment:
This should be changed back.

Reply:
maybe not; im not sure, how the deploy individually would work without it; we should investigate